source/framework/vx_pyramid.c

```diff
@@ -575,57 +579,54 @@ static vx_status ownDestructPyramid(vx_reference ref)
     vx_uint32 i = 0;
     tivx_obj_desc_pyramid_t *obj_desc = NULL;
 
-    if(ref->type == (vx_enum)VX_TYPE_PYRAMID)
+    /* status set to NULL due to preceding type check */
```

Comment:
After removing type check, this comment needs to be updated:
@lcweaver , should it be:

updated to "status check set to NULL due to guaranteed type in internal function" or
removed and NULL changed to non-NULL.

Reply:
done